services/basic_types.proto

```diff
@@ -110,6 +110,7 @@ message AccountID {
          */
         bytes alias = 4;
 
```

Comment:
There should be some docs around this including what it is, size, how it's generated, whether it supports shard.realm.num format or only keccak hash, etc. Also, will the `alias` field still allow 20 byte EVM addresses or reject?

Reply:
So currently if an account has an EC key, we calculate an `EVM address` alias, for example:
Alice's account:
Hedera account id = 0.0.1932
Admin key = EC key
EVM address = `0x71C7656EC7ab88b098defB751B7401B5f6d8976F` -> currently only stored in the in-memory map on the consensus node

The EVM address can be used in a couple of cases, for example in `AccountAmounts` constructed for a crypto transfer.

So we would probably want to continue accepting EVM address values for the alias at least in these cases where it's currently allowed in order to be backward compatible?

Apart from that, I would suggest defining the fields like this:
- `alias` can only be either ECDSA or ED key alias
- `evm_address` can only be the calculated EVM address or Virtual address added to an account

Reply:
We'd basically pull the 20 byte section description out of the alias above and use it here

Reply:
Updated.